Makefile

```diff
@@ -125,15 +125,33 @@ builder: static static-linux-amd64
 
 .PHONY: unit-tests
 unit-tests:
-	go test -v ./...
+	go test -v $(shell go list ./... | grep -v /e2e)
```

Comment:
would be cool to use build tags https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags

Reply:
Yeah, that would be nice but it just breaks my gopls. I will let this out for now, until I figure out how to get gopls properly working here.